docs/extends.md

```diff
@@ -233,11 +233,9 @@ manually keep both environments in sync.
 ### Reference
 
 You can use `extends` on any service together with other configuration keys. It
-always expects a dictionary that should always contain two keys: `file` and
-`service`.
+always expects a dictionary that should always contain the key: `service` and optionally the `file` key.
 
-The `file` key specifies which file to look in. It can be an absolute path or a
-relative one&mdash;if relative, it's treated as relative to the current file.
+The `file` key specifies which file to look for the service in. If it's not set, Compose will look in the current file. When setting the `file` key, it can be an absolute path or a relative one&mdash;if relative, it's treated as relative to the current file.
```

Comment:
The `file` is a bit ambiguous ---  any arbitrary file or a compose configuration file? Also what does it define the service or the extension---either could work.  Also, avoid future tense.

So, a rewrite would look like this:

The `file` key specifies the location of a Compose configuration file defining the extension.  The `file` value can be an absolute or relative path.  If you specify a relative path, Docker Compose treats it as relative to the location of the current file. If you don't specify a `file`, Compose looks in the current configuration file.  

### Editorial guidance on emdash
![image](https://cloud.githubusercontent.com/assets/1347209/8549469/c750ba48-247f-11e5-83fb-3e162b3874a0.png)
![image](https://cloud.githubusercontent.com/assets/1347209/8549478/d5f70110-247f-11e5-870d-c3c13c86c07d.png)